### ISSUE TITLE
KOReader.png does nothing

### ISSUE THREAD

**Issue by FrederickGIT** [koreader/koreader#4005]
* KOReader version: koreader-beta-20180602
* Device: KOBO Aura One Limited Edition

#### Issue - I used the KFMon method, as I find having to restart the device cumbersome. I followed the instructions and the device reacted in the appropriate way. I can not launch KOReader via the .png, it simply does nothing besides opening the image.

I have already modded my device with a few things, including KSM9, so some of that may be interfering.

**Comment by Frenzie** [comment]
Pinging @NiLuJe

**Comment by cramoisi** [comment]
[from mobileread kfmon thread](https://www.mobileread.com/forums/showthread.php?t=274231)

>Of course, running both fmon & kfmon in parallel is highly discouraged. Thankfully, unless you're doing something wrong, you actually can't .

**Comment by NiLuJe** [comment]
It *shouldn't*, AFAICT, but it touches so many moving parts that here be dragons, basically.

So, quick question: Which FW version are you running, and do you have shell access to check a few things?

**Comment by NiLuJe** [comment]
NOTE: I'm working on being able to provide visible on-screen feedback when things go wrong with KFMon, to make these kinds of issues easier to deal with ;).

**Comment by FrederickGIT** [comment]
If you mean the Kobo's software, I'm running 4.8.11073, patched with a few extra functionalities (from here https://www.mobileread.com/forums/showthread.php?t=260100 )
I do not know what shell access actually IS, but I can probably find out if you need me to.

**Comment by NiLuJe** [comment]
Okay, so, to recap:

* The patches will not affect KFMon in any way.
* No issues with your FW version, either.
* AFAICT, KSM won't mess with KFMon either (plus, I've had other people with both confirm that fact).

TL;DR: It's probably something trivial. Looking at the log would probably shed some light on this, but it's not easily accessible without shell access. I won't bother you with this given your answer, don't worry about it ;).

I'm getting closer to actually getting [FBInk](https://github.com/NiLuJe/FBInk) integrated with KFMon, so, hopefully, when I'm done, you'll have a better idea of what went wrong without having to get your hands dirty ;).

**Comment by cramoisi** [comment]
@NiLuJe  rt : what if the png hasn't been correctly processed by nickel before using kfmon with it ? (seems unlikely though because of the install of ksm before (but was it before ?) as the thing is stated as a step of installation)

**Comment by NiLuJe** [comment]
In that case, yeah, it would be prevented to start until fully processed. But the install process should have prevented that if done correctly ;).

**Comment by NiLuJe** [comment]
And that's one of the thing that currently end up in the logs, and that will be accompanied by an on screen message.

**Comment by NiLuJe** [comment]
Woops, tshering pointed out to me that, yes, KSM does inhibit both fmon and KFMon if it starts on boot.

I misread the diff when I was checking this fact, my bad ;).

TL;DR: You'll have to setup your device to boot into Nickel, and not KSM.

**Comment by FrederickGIT** [comment]
Hmm. I don't know how to do that, lwt me see.
My device currently boots into that boot selection screen, where I can pick various boot options? Dunno what it is called. Please excuse my abysmal knowledge of these matters.

On Jun 10, 2018 07:45, NiLuJe <notifications@github.com> wrote:

Woops, tshering pointed out to me that, yes, KSM does inhibit both fmon and KFMon if it starts on boot.

I misread the diff when I was checking this fact, my bad ;).

TL;DR: You'll have to setup your device to boot into Nickel, and not KSM.

**Comment by NiLuJe** [comment]
And I've never used KSM, so, I can't be of much help there ;).

**Comment by KenMaltby** [comment]
Basically, you need to pick one launcher to use.  Clean off your device and start over again.

**Comment by NiLuJe** [comment]
tshering helpfully provided a quick rundown of how to handle this kind of situation [in this post](https://www.mobileread.com/forums/showpost.php?p=3704807&postcount=327) ;).
